Support/requesting-a-public-dns-ptr-record.md

```diff
@@ -19,10 +19,11 @@
 </p>
 <p>&nbsp;&nbsp;Open a support ticket via the portal or by sending email to help@ctl.io.
   <br />&nbsp; In the body, please include the following:
+  <br />&nbsp; - Please include your PIN https://www.ctl.io/knowledge-base/support/pin-authentication-for-support-requests/
```

Comment:
change to <a href="../Support/pin-athentication-for-support-requests.md">PIN authentication</a>